Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.04
```

Comment:
Why not use latest version?

Reply:
Latest version is less stable than 16.04, If you want I can use it

Reply:
No that makes sense. Thanks